### PR TITLE
RHIDP-939: Fix typos and consistency in Getting Started with RHDH doc

### PR DESCRIPTION
**Pull request by hmanwani-rh**
Jira: https://issues.redhat.com/browse/RHIDP-939

Removal of ArtifactHUB: https://issues.redhat.com/browse/RHIDP-1103

### CODE REVIEW

File: getting-started/proc-add-source-control-rhdh-catalog.adoc [hunk 1]
```diff
@@ -77,7 +77,7 @@ To access the GitHub security insights widget on the Overview page in {product-s
 --
 
 . Navigate to the *Helm* tab and select *Upgrade*.
-. Under *Root Schema → Backstage Chart Schema → Backstage Parameters → Backstage container environment variables from Secrets*, add `rhdh-secrets` as value.
+. Under *Root Schema → Backstage Chart Schema → Backstage Parameters → Backstage container environment variables from existing Secrets*, add `rhdh-secrets` as value.
```
Comment (debsmita1) [inline getting-started/proc-add-source-control-rhdh-catalog.adoc:80]: "... from secrets" was correct

Screenshot 2024-01-25 at 5 25 42 PM

Comment (hmanwani-rh) [reply getting-started/proc-add-source-control-rhdh-catalog.adoc:80]: The screenshot in Google doc is outdated. The updated screen shows "existing" in the field name: 
image

**Comment by josephca** [comment]
@hmanwani-rh I added a comment in https://issues.redhat.com/browse/RHIDP-939 

Please take a look at the discussion on https://issues.redhat.com/browse/RHIDP-978 for this part. 

- YAML snippets need to be formatted better

The current snippet has an error and the correction in https://issues.redhat.com/browse/RHIDP-978 looks to be correct.

All other changes in this PR look good though. Thanks,

**Review by josephca — CHANGES_REQUESTED** [review]
This has to be changed to 'providerId'

File: getting-started/proc-add-custom-app-file-openshift.adoc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [id='proc-add-custom-app-file-openshift_{context}']
 = Adding a custom application configuration file to Red Hat OpenShift
 
-To access the {product}, you must a custom application configuration file to OpenShift. In OpenShift, you can use the following content as a base template to create ConfigMap named `app-config-rhdh`:
+To access the {product}, you must add a custom application configuration file to OpenShift. In OpenShift, you can use the following content as a base template to create ConfigMap named `app-config-rhdh`:
```
Comment (coreydaley) [inline getting-started/proc-add-custom-app-file-openshift.adoc:4]:
```suggestion
To access the {product}, you must add a custom application configuration file to OpenShift. In OpenShift, you can use the following content as a base template to create a ConfigMap named `app-config-rhdh`:
```

File: installation/proc-install-rhdh-helm.adoc [hunk 3]
```diff
@@ -10,7 +10,7 @@ Helm is a package manager on OpenShift that provides the following features:
 * Provides charts that you can host on public and private servers
 * Supports rolling back to previous application versions
 
-The {product} Helm chart is available in the Helm catalog in Red Hat OpenShift Dedicated, OpenShift Container Platform (OCP) or on ArtifactHUB.
+The {product} Helm Chart is available in the Helm catalog in Red Hat OpenShift Dedicated, OpenShift Container Platform (OCP) or on ArtifactHUB.
```
Comment (coreydaley) [inline installation/proc-install-rhdh-helm.adoc:13]:
```suggestion
The {product} Helm Chart is available in the Helm catalog on Red Hat OpenShift Dedicated, OpenShift Container Platform (OCP) or on ArtifactHUB.
```

**Review by coreydaley — COMMENTED** [review]
A couple of small suggestions

File: getting-started/proc-add-custom-app-file-openshift.adoc (hunk 2 above)
Comment (hmanwani-rh) [reply getting-started/proc-add-custom-app-file-openshift.adoc:4]: Done

File: installation/proc-install-rhdh-helm.adoc (hunk 3 above)
Comment (hmanwani-rh) [reply installation/proc-install-rhdh-helm.adoc:13]: Done

Comment (kim-tsao) [inline installation/proc-install-rhdh-helm.adoc:13]: Please remove ArtifactHub here as well

Comment (hmanwani-rh) [reply installation/proc-install-rhdh-helm.adoc]: Done

**Review by Gerry-Forde — APPROVED** [review]
LGTM

**Pull request merged**